models/model-cards/gnn-fsi-model-card.md

```diff
@@ -0,0 +1,228 @@
+
+**Dataset License:**  
+* [Apache 2.0](http://www.apache.org/licenses/LICENSE-2.0) <br>
+
+## Evaluation Dataset:
+**Link:**  
+* [fraud-detection-validation-data.csv](models/dataset/fraud-detection-validation-data.csv)  <br>
+
+**Properties (Quantity, Dataset Descriptions, Sensor(s)):** 
+* Data consists of raw 265 labeled credit card transaction synthetically created<br>
+
+**Dataset License:**
+*  [Apache 2.0](http://www.apache.org/licenses/LICENSE-2.0)<br>
+
+## Inference:
+**Engine:** 
+* Triton <br>
+
+**Test Hardware:** <br>
+* Other   <br>
+
+# Subcards
+## Model Card ++ Bias Subcard
+
+### What is the gender balance of the model validation data?  
+* Not Applicable
+
+### What is the racial/ethnicity balance of the model validation data?
+* Not Applicable
+
+### What is the age balance of the model validation data?
+* Not Applicable
+
+### What is the language balance of the model validation data?
+* Not Applicable
+
+### What is the geographic origin language balance of the model validation data?
+* Not Applicable
+
+### What is the educational background balance of the model validation data?
+* Not Applicable
+
+### What is the accent balance of the model validation data?
+* Not Applicable
+### Describe measures taken to mitigate against unwanted bias.
+* Not Applicable
+## Model Card ++ Explainability Subcard
+
+### Name example applications and use cases for this model. 
+* This model is intended to be used for testing fraud detection use cases in morpheus pipeline using application of GNN.
+### Fill in the blank for the model technique.
+* This model is intended for developers that want to build and/or customize fraud detection applications using graph neural network.
+
+### Name who is intended to benefit from this model. 
+* This model is intended for users that use transactional data to build fraud detection application.
+
+### Describe the model output.
+* This model outputs fraud probability score b/n (0 & 1). 
+
+### List the steps explaining how this model works. (e.g., )  
+* The model uses a bipartite heterogeneous graph representation as input for `GraphSAGE` for feature learning and `XGBoost` as a classifier. Since the input graph is heterogeneous, a heterogeneous implementation of `GraphSAGE` (HinSAGE) is used for feature embedding.<br>
+
+### Name the adversely impacted groups (protected classes) this has been tested to deliver comparable outcomes regardless of:
+* Not Applicable
+
+### List the technical limitations of the model.
+* This model version requires a transactional data schema with entities (user, merchant, transaction) as requirement for the model.
+
+### What performance metrics were used to affirm the model's performance?
+* Area under ROC curve and Accuracy
+
+### What are the potential known risks to users and stakeholders? 
+* None
+
+### What training is recommended for developers working with this model?  If none, please state "none."
+* none
```

Comment:
Let's update this to say `Familiarity with the Morpheus SDK is recommended for developers working with this model.`